modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,150 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with manually with the following procedure.
+
+.Procedure
+
+. Retrieve the {product-title} client (`oc`):
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz > oc.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxf oc.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x oc
+----
+
+. Set the {product-title} version:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version>
+----
+
+. Retrieve the {product-title} installer:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-$OCP_VERSION/openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxvf openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x openshift-install
+----
+
+. Retrieve the {op-system-first} ISO:
++
+[source,terminal]
+----
+$ ISO_URL=$(openshift-install coreos print-stream-json | \
+          jq .architectures.x86_64.artifacts.metal.formats.iso.disk.location
+----
++
+[source,terminal]
+----
+$ curl $ISO_URL > rhcos.iso
+----
+
+. Prepare the installation file:
++
+[source,terminal]
+----
+apiVersion: v1
+baseDomain: <domain> <1>
+compute:
+- name: worker
+  replicas: 0 <2>
+controlPlane:
+  name: master
+  replicas: 1 <3>
+metadata:
+  name: <name> <4>
+networking:
+  networkType: OVNKubernetes
+  clusterNetwork:
+  - cidr: <IP_address>/<prefix> <5>
+    hostPrefix: <prefix> <6>
+  serviceNetwork:
+  - <IP_address>/<prefix> <7>
+platform:
+  none: {}
+BootstrapInPlace:
+  InstallationDisk: <path_to_install_drive> <8>
+pullSecret: '<pull_secret>' <9>
+sshKey: |
+  <ssh_key> <10>
+----
++
+<1> Add the cluster domain name.
++
```

Comment:
I don't think you need the `+` between these callouts

Reply:
They are within a numbered list, that's why they are there. Otherwise, they would represent a new line.